x-pack/plugins/data_visualizer/public/application/common/components/not_in_docs_content/not_in_docs_context.tsx

```diff
@@ -6,18 +6,15 @@
  */
 
 import React, { FC, Fragment } from 'react';
-import { EuiIcon, EuiSpacer, EuiText } from '@elastic/eui';
+import { EuiIcon, EuiText } from '@elastic/eui';
 
 import { FormattedMessage } from '@kbn/i18n/react';
 
 export const NotInDocsContent: FC = () => (
   <Fragment>
-    <EuiSpacer size="xxl" />
     <EuiText textAlign="center">
       <EuiIcon type="alert" />
     </EuiText>
-
-    <EuiSpacer size="s" />
     <EuiText textAlign="center">
```

Comment:
Did you try setting the size to `xs` for the warning message here? It makes it consistent with the other text in the rows:

![image](https://user-images.githubusercontent.com/7405507/135287495-03ada706-412d-4318-b0ea-79f4724a53db.png)

compared to the default of `m`:
![image](https://user-images.githubusercontent.com/7405507/135287840-c184e71d-bb33-4243-b0e2-50d484cdfc7c.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/112870/commits/4562aa87dc97ca1e23cfb573a55e0399b307d55e